cudamapper/tests/Test_CudamapperOverlapperTriggered.cu

```diff
@@ -169,17 +169,18 @@ TEST(TestCudamapperOverlapperTriggerred, OneAchorNoOverlaps)
     testv.push_back("READ2");
     std::vector<std::uint32_t> test_read_length(testv.size(), 1000);
 
-    EXPECT_CALL(test_index, read_id_to_read_name)
-        .WillRepeatedly(testing::ReturnRef(testv));
-    EXPECT_CALL(test_index, read_id_to_read_length)
-        .WillRepeatedly(testing::ReturnRef(test_read_length));
+    for (std::size_t i = 0; i < testv.size(); ++i)
```

Comment:
Optional: Just because I would like to establish more consistent style: could you use the following?
```
for (std::int64_t i = 0; i < get_size(testv); ++i)
```
(also in the following loops)

Reply:
I generally agree with the idea of using signed values for loops, but let's do this in a separate PR (there's a lot of signed-unsigned things we should address)

Reply:
I just meant because you edit this loop anyways. I didn't want you to fix the other loops, but ok, let's do a separate PR.